schema.json

```diff
@@ -0,0 +1,496 @@
+                                                    "name": {
+                                                        "type": "string"
+                                                    },
+                                                    "bin_name": {
+                                                        "type": "string"
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "apt_get": {
+                                            "type": "array",
+                                            "items": {
+                                                "type": "object",
+                                                "additionalProperties": false,
+                                                "properties": {
+                                                    "name": {
+                                                        "type": "string"
+                                                    },
+                                                    "bin_name": {
+                                                        "type": "string"
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "check_only": {
+                                            "type": "array",
+                                            "items": {
+                                                "type": "object",
+                                                "additionalProperties": false,
+                                                "required": [
+                                                    "name"
+                                                ],
+                                                "properties": {
+                                                    "name": {
+                                                        "type": "string"
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                },
+                                "is_requires_admin_user": {
+                                    "type": "boolean"
+                                },
+                                "is_always_run": {
+                                    "type": "boolean"
+                                },
+                                "is_skippable": {
+                                    "type": "boolean"
+                                },
+                                "run_if": {
+                                    "type": "string"
```

Comment:
some kind of format validation? for example can we run the cli's run_if parser on it? (I saw custom validators for this go dep)

Reply:
i am not sure if we need this, it is like testing if an input in the step.yml is used in the source code.